docs/gettinginvolved/infrastructure/release-process.rst

```diff
@@ -290,6 +265,15 @@ Release
 
    Wait for the build to complete successfully.
 
+#. Set up ``AWS Access Key ID`` and ``AWS Secret Access Key`` Amazon S3 credentials:
```

Comment:
I wonder if it makes sense to but per-requisite things like this in a section on its own at the very beginning. This step should only need to be done once per person.